### PR TITLE
instrument: refactor, format, and organize code by integration

### PR DESCRIPTION
**Pull request by darccio**
### What does this PR do?

Refactor duplicated code by reusing utility functions like `wrap` from adding `dd:startwrap` and `dd:endwrap`, clean format by removing some blank lines, and organize integrations' code in separate files. Also, all `labstack/echo.v4` code uses `[eE]choV4` as prefix.

### Motivation

`instrument.go` and `uninstrument.go` were getting really big.

### Reviewer's Checklist

- [ ] Changed code has unit tests for its functionality.

### CODE REVIEW

**Review by ahmed-mez — APPROVED** [review]
🚀

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .DEFAULT_GOAL := build
 
-.PHONY: build generate vet test clean fmt licenses verify-licenses
+.PHONY: build generate tidy vet test clean fmt licenses verify-licenses
```
Comment (darccio) [inline Makefile:3]: Question: what is the purpose of `generate` target? I saw it only declared as a phony target, but it doesn't seem to have any use. cc @ahmed-mez @knusbaum @jonbodner

Comment (ahmed-mez) [reply Makefile:3]: It's likely an oversight when we first created the Makefile? Perhaps it was copied from another Go project that has a generate target. As far as Orchestrion is concerned, we don't have any code generation yet. You can get rid of it if you want.

Comment (darccio) [reply Makefile:3]: As it's a minor nit, I'll keep it. Maybe the idea was to run `go:generate` commands? I see one usage of `stringer` in `event.go.`

**Pull request merged**